### PR TITLE
Enhancement: Contact documentation

### PR DESCRIPTION
**Pull request by BenjamenMeyer**
This contributes to the following stories:
https://github.com/vegastrike/infrastructure-tracking/issues/1
https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/96
https://github.com/vegastrike/infrastructure-tracking/issues/6

Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [x] This is a documentation change only

Issues:
- https://github.com/vegastrike/infrastructure-tracking/issues/1
- https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/96
- https://github.com/vegastrike/infrastructure-tracking/issues/6
- https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/178

Purpose:
Document our means of contacts, submitting bugs, etc where it'll get passed out with releases, source, etc.
Rename binaries per naming convention in #178.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -610,6 +610,31 @@ Submit comments or suggestions by opening an [issue](https://github.com/vegastri
 
 And if you can design some missions it would rock!
 
+Vega Strike Contacts
+====================
+
+Vega Strike is the product of many contributors from all around the world. If you need help, find a bug, want to request a features, etc then please contact us all using one
```
Comment (stephengtuggy) [inline README.md:616]: Should be `feature` rather than `features`

**Review by stephengtuggy — CHANGES_REQUESTED** [review]
Apart from one small typo, everything looks good.

File: README.md (hunk 1 above)
Comment (BenjamenMeyer) [reply README.md]: fixed, thanks for catching that!

**Comment by BenjamenMeyer** [comment]
Note: Packaged 6a6a8f5 in here because of already packaging c7a6e99 and the discussion in #178 related to c7a6e99 resolved by https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/178#issuecomment-769155620. Therefore this should be able to close out #178 in addition to the other issues once it's backported to master.

**Review by stephengtuggy — APPROVED** [review]
Looks good to me!

**Pull request merged**